### PR TITLE
remove std log.

### PR DESCRIPTION
**Pull request by FishMeat**
#38

### CODE REVIEW

File: simulator/consensus_sim_server.go [hunk 1]
```diff
@@ -197,23 +196,23 @@ func main() {
 	if *cpuprofile != "" {
 		f, err := os.Create(*cpuprofile)
 		if err != nil {
-			log.Fatal(err)
+			logger.Fatal().Err(err)
```
Comment (zjshen14) [inline simulator/consensus_sim_server.go]: I think zerolog needs Msg/Msgf to make the log be fired

File: simulator/consensus_sim_server.go [hunk 2]
```diff
@@ -197,23 +196,23 @@ func main() {
...
 		}
 	}
 
 	lis, err := net.Listen("tcp", port)
 	if err != nil {
-		log.Fatalf("failed to listen: %v", err)
+		logger.Fatal().Msgf("failed to listen: %v", err)
```
Comment (zjshen14) [inline simulator/consensus_sim_server.go]: Use Err(err) in addition to msg

File: tools/addrgen/internal/cmd/createconfig.go [hunk 3]
```diff
@@ -8,8 +8,8 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/iotexproject/iotex-core/logger"
```
Comment (zjshen14) [inline tools/addrgen/internal/cmd/createconfig.go]: group the imports in the way:
```
import (
    std

    thirdparty

    firstparty
)
```

File: tools/addrgen/internal/cmd/generate.go [hunk 4]
```diff
@@ -21,7 +21,7 @@ var generateCmd = &cobra.Command{
 	Short: "Generates n number of iotex address key pairs.",
 	Long:  `Generates n number of iotex address key pairs.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		log.Printf("\n%s\n", generate(args))
+		logger.Logger().Printf("\n%s\n", generate(args))
```
Comment (zjshen14) [inline tools/addrgen/internal/cmd/generate.go]: Please avoid use printf. Use info

**Comment by FishMeat** [comment]
@zjshen14 modifyed

File: simulator/consensus_sim_server.go [hunk 5]
```diff
@@ -197,23 +196,23 @@ func main() {
 	if *cpuprofile != "" {
 		f, err := os.Create(*cpuprofile)
 		if err != nil {
-			log.Fatal(err)
+			logger.Fatal().Err(err).Msgf("failed to create file: %v", err)
```
Comment (zjshen14) [inline simulator/consensus_sim_server.go]: Since you already add error to the error field. You don't need to add it to msg again. Same for the following.

Comment (FishMeat) [reply simulator/consensus_sim_server.go]: ok

**Pull request merged**